external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -1991,8 +1981,8 @@ fn parse_name_exp(context: &mut Context) -> Result<Exp_, Box<Diagnostic>> {
 }
 
 // Parse the arguments to a call: "(" Comma<Exp> ")"
-fn parse_call_args(context: &mut Context) -> Result<Spanned<Vec<Exp>>, Box<Diagnostic>> {
-    ok_with_loc!(
+fn parse_call_args(context: &mut Context) -> Spanned<Vec<Exp>> {
+    let (loc, args) = with_loc!(
```

Comment:
Not sure what I am supposed to do here... Both `ok_with_loc!` and `with_loc!` were there before and used one instead of the other as this no longer returns `Ok`

Reply:
I was thinking about somethinkg like `sp_with_loc`, but this is good enough, I agree.